roles/matrix-bot-matrix-registration-bot/tasks/validate_config.yml

```diff
@@ -0,0 +1,9 @@
+---
+
+- name: Fail if required settings not defined
+  fail:
+    msg: >-
+      You need to define a required configuration setting (`{{ item }}`).
+  when: "vars[item] == ''"
+  with_items:
+    - "matrix_bot_matrix_registration_bot_api_token"
```

Comment:
It's probably best to validate both `matrix_bot_matrix_registration_bot_bot_access_token` and `matrix_bot_matrix_registration_bot_api_token` here.

Reply:
Done

Reply:
Removed the file but changed the variable name accordingly